crates/llms/Cargo.toml

```diff
@@ -33,7 +33,11 @@ tracing.workspace = true
 tracing-futures.workspace = true
 
 ## For Chunking
-text-splitter = {  git = "https://github.com/Jeadie/text-splitter.git", rev = "b33e4748576b9a81d436d3f28a6706f860fc45ba", features = ["markdown", "tokenizers", "tiktoken-rs"]}
+text-splitter = { git = "https://github.com/Jeadie/text-splitter.git", rev = "b33e4748576b9a81d436d3f28a6706f860fc45ba", features = [
```

Comment:
```suggestion
text-splitter = { git = "https://github.com/spiceai/text-splitter.git", rev = "b33e4748576b9a81d436d3f28a6706f860fc45ba", features = [
```

Reply:
I've created the fork and pushed commit `b33e4748576b9a81d436d3f28a6706f860fc45ba` to the `spiceai` branch